hosts.go

```diff
@@ -295,6 +295,15 @@ func (c *Client) RetireHost(id string) error {
 	return err
 }
 
+// BulkRetireHosts retires the hosts
+func (c *Client) BulkRetireHosts(ids []string) error {
```

Comment:
`POST /api/v0/hosts/bulk-retire` endpoint places a limit on the number of hosts to retire. It is more careful to note the limitation in the function comments. (It is not necessary to write a specific number, as the limit may change.)